<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraCalibrationSidePanelWidget</class>
 <widget class="QWidget" name="CameraCalibrationSidePanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Current camera</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="currentCameraComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="calibrationGridGroupBox">
     <property name="title">
      <string>Calibration Grid</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="gridWidthSpinBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="gridHeightSpinBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Square size (mm):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="gridSquareSizeSpinBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="showGridCheckBox">
        <property name="text">
         <string>Show grid</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="capturedViewsGroupBox">
     <property name="title">
      <string>Captured Views</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="showAllCapturedViewsButton">
          <property name="text">
           <string>Show all</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hideAllCapturedViewsButton">
          <property name="text">
           <string>Hide all</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="extrinsicScaleGroupBox">
     <property name="title">
      <string>Extrinsic optim. scale</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Translation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="translationScaleSpinBox">
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Rotation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="rotationScaleSpinBox">
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="updateCalibrationButton">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="extrinsicReprojErrorLabel">
     <property name="text">
      <string>Avg. ext. reproj. error: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="calibrateButton">
     <property name="text">
      <string>Capture Views...</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exportCalibrationDataButton">
     <property name="text">
      <string>Export data...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="importDataButton">
     <property name="text">
      <string>Import data...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="validationButton">
     <property name="text">
      <string>Cross Validation...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="viewListWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
